packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -55,6 +55,10 @@ import { FormatType } from './saveable';
 import { QuickInputService, QuickPick, QuickPickItem } from './quick-input';
 import { AsyncLocalizationProvider } from '../common/i18n/localization';
 import { nls } from '../common/nls';
+import { confirmExit } from './dialogs';
+import { WindowService } from './window/window-service';
+import { ConfirmDialog, Dialog } from '.';
```

Comment:
Minor: We shouldn't import from the index.

Reply:
Thanks, good catch. Between `src` imports and `.` imports, auto-import functionality is almost more trouble than it's worth. 🤦‍♂️